packages/flutter_tools/lib/executable.dart

```diff
@@ -85,7 +85,7 @@ Future<Null> main(List<String> args) async {
     ..addCommand(new UpdatePackagesCommand(hidden: !verboseHelp))
     ..addCommand(new UpgradeCommand());
 
-  return Chain.capture(() async {
+  return Chain.capture/*<Future<Null>>*/(() async {
```

Comment:
@leafpetersen, @jmesserly: needing this seems odd.  Shouldn't inferencing make it unnecessary?